tests/karate/src/test/java/fe/utils/web.feature

```diff
@@ -1,6 +1,7 @@
 Feature: web test
 
   Background: App Preset
+    #* configure driver = { type: 'chrome', executable: 'C:/Program Files/Google/Chrome/Application/chrome.exe'}
```

Comment:
I think this should not be present here, if it will be used in an upcoming PR better place it there

Reply:
This is on purpose. Karate considers that the default installation file for Chrome is `/Program Files(x86)/` but another common default file is `/Program Files/` from what I've read Google went a bit back and forth on this and furthermore it depends if the installation is for all users or not. 
So the idea of the commented line is to be able to quickly change for people that need it.

Reply:
Okay I agree than